messages/rest.md

```diff
@@ -81,4 +81,4 @@ HTTP header in "key:value" format.
 
 # flags.body.summary
 
-File or content for the body of the HTTP request. Specify "-" to read from standard input or "" for an empty body.
+File or content for the body of the HTTP request. Specify "-" to read from standard input or "" for an empty body. If passing a file, prefix it with '@' like CURL
```

Comment:
```suggestion
File or content for the body of the HTTP request. Specify "-" to read from standard input or "" for an empty body. If passing a file, prefix it with '@', similar to --data flag of curl. 
```

Reply:
I don't think we need to mention curl.  Especially because there's an example, and you added the note.